sources/api/ecs-settings-applier/src/ecs.rs

```diff
@@ -87,6 +93,8 @@ async fn run() -> Result<()> {
             .map(|s| s.to_string())
             .collect(),
         spot_instance_draining_enabled: ecs.enable_spot_instance_draining,
+        warm_pools_support: ecs.warm_pools_check,
+        image_pull_behavior: Some(ecs.image_pull_behavior.unwrap().inner_code.parse::<u8>().unwrap()),
```

Comment:
With the `TryInto` suggestion I made elsewhere, this would become something like:
```suggestion
        image_pull_behavior: ecs.image_pull_behavior.try_into() as u8,
```